autogen/main/outputs.tf.tmpl

```diff
@@ -76,6 +76,24 @@ output "endpoint" {
   ]
 }
 
+output "endpoint_dns" {
+  sensitive   = true
```

Comment:
@apeabody I have a question about the sensitivity. I know the `endpoint` output is also marked as sensitive, but do you think it makes sense for both outputs? It's marked as sensitive from the initial commit to this module without any description. I know the standard way of configuring kubectl is via `gcloud container clusters get-credentials`, but should the endpoint IP/DNS be really sensitive? 🤔

Reply:
Hi @gorge511 - I carried it over, but had the same through it wasn't really required (it's not a token, cert, password, etc).  I'm considering to remove from both.

Reply:
removed from endpoint_dns for now